vendor/bat-native-ledger/src/bat/ledger/internal/bat_contribution.cc

```diff
@@ -1829,4 +1837,117 @@ void BatContribution::DoRetry(const std::string& viewing_id) {
   }
 }
 
+void BatContribution::OnRemovePendingContribution(ledger::Result result) {
+  if (result == ledger::Result::LEDGER_OK) {
+    ledger_->OnContributeUnverifiedPublishers(
+        ledger::Result::PENDING_PUBLISHER_REMOVED);
+  }
+}
+
+void BatContribution::OnContributeUnverifiedPublishers(
+    double balance,
+    const ledger::PendingContributionInfoList& list) {
+  if (list.empty()) {
+    return;
+  }
+
+  if (!list.empty() && balance == 0) {
```

Comment:
Seems like you don't need the `!list.empty()` check here, given the check above.

Reply:
removed